content/en/storage-providers/curio/design/index.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Curio Design"
+description: "In-depth explanation of the Curio's design and core components"
+lead: "This page provides a detailed overview of the core concepts and components that make up Curio, including HarmonyDB, HarmonyTask, and more."
+draft: false
+menu:
+    storage-providers:
+        parent: "curio"
+weight: 110
+toc: true
+---
```

Comment:
This page should probably have aliases to the old lotus-provider/developer.md page, to keep links intact.

Reply:
Alias are required to avoid broken links in large docs. There would be no need here as these docs were self contained. I see no link to any of these pages.

Reply:
Sorry, I disagree here. We have links floating around in Slack, and announcements to: https://lotus.filecoin.io/storage-providers/lotus-provider/XXXXXX. And those will break.

It seems like a super small task to add these to preserve the links.

Reply:
@rjan90 Did I miss something?

Reply:
> I wanted to touch the "This week in Lotus" as well but those are governed by Lotus team.

@LexLuthr hrm let me be clear: we are still working in the same repository and delivering software for our users. We are still all lotus maintainers. The plan is to maintain lotus miner and boost for network users UNTIL there is proven adoption of the new curio stack. I am very excited about the new stack, however, its not even GA yet. This is way too early to not maintain good quality  & good ux of lotus miner. Have clear access to documentation is one of them.

Reply:
@jennijuju Let me clarify these link are not used internally in the docs anywhere except "This Week in Lotus". We can fix those pages to point to new docs easily. Now, the only use case left here is keeping some links in Slack alive.  I just don't see any upside to it except a handful of people might go back to those posts and use those links. 
I will pushing a new PR to add the aliases.